server/service.go

```diff
@@ -149,23 +158,23 @@ func NewService() (svr *Service, err error) {
 		}
 		var l net.Listener
 		if httpMuxOn {
-			l = svr.muxer.ListenHttp(0)
+			l = svr.muxer.ListenHttp(1)
 		} else {
 			l, err = net.Listen("tcp", address)
 			if err != nil {
 				err = fmt.Errorf("Create vhost http listener error, %v", err)
 				return
 			}
+			go server.Serve(l)
 		}
-		go server.Serve(l)
```

Comment:
This can not be deleted. and should delete the line 168. Otherwise when the bind_port == v_http_port,  I don't find the http server.Serve! the result of test is as my excepted